src/config/dynamic-theme-fixes.config

```diff
@@ -367,6 +367,16 @@ CSS
 
 ================================
 
+accounts.firefox.com
```

Comment:
We generally don't make any fixes for protected websites/websites in the dark list.